File: 9/solution.js
console.log((function getPythagoreanTripleProduct(sum) {
	for (var a = 1; a < sum; a += 1) {
		for (var b = 1; b < sum; b += 1) {
			c = sum - a - b;
			if (Math.pow(a, 2) + Math.pow(b, 2) === Math.pow(c, 2)) {
				return a * b * c;
			}
		}
	}
})(1000));

File: 9/solution.rb
def getPythagoreanTripleProduct sum
	(1..sum).each do |a|
		(1..sum).each do |b|
			c = sum - a - b
			if (a ** 2) + (b ** 2) == c ** 2
				return a * b * c
			end
		end
	end
end

puts getPythagoreanTripleProduct 1000